setupbase.py

```diff
@@ -398,6 +398,28 @@ def run(self):
         update_package_data(self.distribution)
 
 
+def patch_out_bootstrap_bw_print():
+    """Hack! Manually patch out the bootstrap rule that forces printing in B&W.
+
+    We haven't found a way to override this rule with another one.
+    """
+    print_less = pjoin(static, 'components', 'bootstrap', 'less', 'print.less')
+    with open(print_less) as f:
+        lines = f.readlines()
+
+    for ix, line in enumerate(lines):
+        if 'Black prints faster' in line:
```

Comment:
I guess this is enough specific to patch the proper line, right?

Reply:
Should be. The original file is not very long:

https://github.com/twbs/bootstrap/blob/v3.3.7/less/print.less